### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by dxArtemiusz**
Updated general README
Added additional platform specific info into platform READMEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,23 @@
 
 # Selector for DevExtreme - How to implement Theme Selector for the DevExtreme Project Template
 
-The DevExtreme Project Template has two default themes: `base` and `swatch`. You can change the base theme to another Material theme at runtime. You can switch between themes using the StyleSheet API. For this, disable all loaded theme stylesheet links excluding the active one.
+The DevExtreme Project Template has two default themes: `base` and `swatch`. You can change the base theme to another Material theme at runtime. You can switch between themes using the StyleSheet API. For this, disable all loaded theme stylesheet links excluding the active one. 
+
```
Comment (sergepilipchuk) [inline README.md:9]: ... Material theme at runtime. To switch between themes, use the StyleSheet API. For this, ...

File: angular/README.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
+# The Angular specific part of Theme Selector implementation
+
+Themes should be declared in the [angular.json](angular/angular.json) file. Also, in this file should be declared an asset folder, where all icons in svg format placed. The optimization configuration for styles should be like following:
```
Comment (sergepilipchuk) [inline angular/README.md]: ... file. In this file, you should declare an asset folder, where all icons store in svg format . In the example below, you can see the optimization configuration for styles:

File: README.md (hunk 1 above)
Comment (dxArtemiusz) [reply README.md:9]: Resolved

File: angular/README.md (hunk 2 above)
Comment (dxArtemiusz) [reply angular/README.md]: resolved

**Pull request merged**